### PR TITLE
[Tools] Make one tool

### PR DESCRIPTION
**Pull request by zhangchiqing**
Now we just need to run `make docker-all-tools` to build all tools.

In the future, we will merge util, read-badger, read-protocol-state into one tool. And remove-execution-fork and remove-execution-results in its own tool

### CODE REVIEW

File: network/test/epochtransition_test.go [hunk 1]
```diff
@@ -127,6 +127,7 @@ func (t *testNodeList) networks() []*p2p.Network {
 }
 
 func TestEpochTransitionTestSuite(t *testing.T) {
+	t.Skip("flaky")
```
Comment (zhangchiqing) [inline network/test/epochtransition_test.go:130]: [Getting this error on CI ](https://github.com/onflow/flow-go/pull/417/checks?check_run_id=1920863056)
```
{"level":"error","error":"could not generate topology: could not generate fanout for topic sync-cluster-flow-testnet: failed to find cluster peers for node e121b1728cdefec4de6b4218b2d2beb0ec6f0ee90e90a967e51bc7a9eb4a2ef3: failed to find the cluster for node ID e121b1728cdefec4de6b4218b2d2beb0ec6f0ee90e90a967e51bc7a9eb4a2ef3","message":"failed to update peers"}
--- FAIL: TestEpochTransitionTestSuite (11.67s)
    --- FAIL: TestEpochTransitionTestSuite/TestNodesAddedAndRemoved (7.05s)
        epochtransition_test.go:316: 
            	Error Trace:	epochtransition_test.go:316
            	            				epochtransition_test.go:290
            	Error:      	Condition never satisfied
            	Test:       	TestEpochTransitionTestSuite/TestNodesAddedAndRemoved
            	Messages:   	node is not connected to enough nodes
```

Comment (zhangchiqing) [reply network/test/epochtransition_test.go:130]: Can we let the test to wait until the collection peers are ready? @yhassanzadeh13

Comment (yhassanzadeh13) [reply network/test/epochtransition_test.go:130]: @zhangchiqing yes, but would be great to have @vishalchangrani confirmation on this as well.

File: Makefile [hunk 2]
```diff
@@ -372,6 +372,45 @@ docker-run-access:
...
+docker-all-tools:tool-util tool-read-badger tool-read-protocol-state tool-remove-execution-fork
+
+PHONY: docker-build-util
+docker-build-util:
+	docker build -f cmd/Dockerfile --ssh default --build-arg TARGET=util --target production \
+		-t "$(CONTAINER_REGISTRY)/util:latest" -t "$(CONTAINER_REGISTRY)/util:$(SHORT_COMMIT)" -t "$(CONTAINER_REGISTRY)/util:$(IMAGE_TAG)" .
+
+PHONY: tool-util
+tool-util: docker-build-util
+	docker container create --name util gcr.io/flow-container-registry/util:latest;docker container cp util:/bin/app ./util;docker container rm util
```
Comment (jordanschalm) [inline Makefile:385]:
```suggestion
	docker container create --name util gcr.io/$(CONTAINER_REGISTRY)/util:latest;docker container cp util:/bin/app ./util;docker container rm util
```

Likewise for the other targets.

File: Makefile [hunk 3]
```diff
@@ -372,6 +372,45 @@ docker-run-access:
 docker-run-ghost:
 	docker run -p 9000:9000 -p 3569:3569 "$(CONTAINER_REGISTRY)/ghost:latest" --nodeid 1234567890123456789012345678901234567890123456789012345678901234 --entries ghost-1234567890123456789012345678901234567890123456789012345678901234@localhost:3569=1000
 
+PHONY: docker-all-tools
+docker-all-tools:tool-util tool-read-badger tool-read-protocol-state tool-remove-execution-fork
```
Comment (jordanschalm) [inline Makefile:376]:
```suggestion
docker-all-tools: tool-util tool-read-badger tool-read-protocol-state tool-remove-execution-fork
```

**Review by jordanschalm — APPROVED** [review]

File: network/test/epochtransition_test.go (hunk 1 above)
Comment (vishalchangrani) [reply network/test/epochtransition_test.go:130]: am ok with adding the skip now. @yhassanzadeh13 gave me some suggestions on how this test can be made more stable. I will work on it as part https://github.com/dapperlabs/flow-go/issues/5277

**Comment by zhangchiqing** [comment]
Tests are passing. Need one more approver.

**Review by yhassanzadeh13 — APPROVED** [review]

**Pull request merged**